tests/test_examples.py

```diff
@@ -61,6 +62,126 @@
 
 CACHE_REPO_NAME = "optimum-internal-testing/optimum-neuron-cache-for-testing"
 
+USE_VENV = False  # os.environ.get("USE_VENV", "true") == "true"
+
+
+class TPSupport(str, Enum):
+    """
+    Describes the support for Tensor Parallelism for a given model:
+
+        - full: The model can be fully parallelized (embeddings + blocks + cross-entropy loss when it makes sense).
+        - partial: The model can be parallelized but not the embeddings. Usually because the vocabulary size is not
+        divisible by the tensor parallel size (2 here).
+        - none: The model cannot be parallelized, either for shape mismatch as in the partial case, or because the
+        tensor parallelism support for this model type has not been added.
+    """
+
+    FULL = "full"
+    PARTIAL = "partial"
+    NONE = "none"
+
+
+class Priority(str, Enum):
```

Comment:
Isn't it coverage other than priority?

Reply:
Renamed to coverage, thanks for the suggestion!